tasks/lib/slack_api/methods_spider.rb

```diff
@@ -36,7 +36,9 @@ def process_list(page, _default_data = {})
 
     def process_method(page, default_data = {})
       method_page = ensure!(page, '.apiMethodPage', default_data[:method_name])
-      desc = method_page.search('.apiReference__mainDescription').text.gsub("’", "'")
+      desc = method_page.search('.apiReference__mainDescription').text.gsub('’', "'")
+      return if desc.downcase.start_with? 'deprecated:'
```

Comment:
Shouldn't this change the value of `deprecated: ` below instead?

Reply:
It looks like that used to be the behavior but was changed in #48 to hardcode `'deprecated' => false` and just remove any deprecated methods, so this is an extra check for when `isDeprecated` isn't accurate
https://github.com/slack-ruby/slack-api-ref/blob/f213e377393aed5c466832f087d40cd35d07274e/tasks/lib/slack_api/methods_spider.rb#L15